frontend/src/components/DotsButton/DotsButton.styled.js

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+import {device} from "../../RWD/device";
+
+export const StyledDotsButton = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  top: 0;
+  right: 0;
```

Comment:
```suggestion
  position: absolute;
  top: 0;
  right: 0;
  display: flex;
  justify-content: center;
  align-items: center;
```

Wcześniej stosowałeś takie ułożenie
(najpierw wartości od position i szerokości i wyskości, potem display i jego właściwości